jest/helpers/asserts.js

```diff
@@ -1,5 +1,26 @@
+// ** SNAPSHOT FILTERING ***
+// prevent snapshots for logs that change on each instance, such as shuffled quiz options
+
+const excludedPhrases = [
+  "Cool! Here's what we get when we mix up the order of the answer choices:"
+]
+
+// returns true if log includes a string from the excludedPhrases array
+function isExcluded (log) {
+  for (let phrase of excludedPhrases) {
+    if (log.includes(phrase)) {
+      return true
+    }
+  }
+  return false
+}
+
+// add log to snapshot UNLESS it contains an excluded phrase
 function assertLogSnapshot (log) {
-  expect(log).toMatchSnapshot()
+  if (!isExcluded(log)) {
+    expect(log).toMatchSnapshot() // expect if log isn't excluded from exertion
```

Comment:
I don't understand this comment because of the word exertion. I think it's just a language issue that I don't understand, but just wanted to make sure that this is the case, because I think you meant `assertion` instead of `exertion`?